lib/api/user_token_service.rb

```diff
@@ -3,21 +3,25 @@ class UserTokenService
     def initialize(config = Settings, args = {})
       @config = config
       @svc_options = args
-      @token_mgr = new_token_mgr(base_config[:module], base_config[:name], api_config)
     end
 
-    attr_accessor :token_mgr
+    def token_mgr(namespace = nil)
+      @token_mgr ||= {}
+      if namespace.nil? || namespace == 'ui' # The default token and UI token share the same TokenStore namespace
+        @token_mgr['ui'] ||= new_token_mgr(base_config[:module], base_config[:name], api_config)
```

Comment:
Will add `api`, but we need to support `nil` and `api` For backward compatibility.

The user can request `GET /api/auth` and get the standard-lived token (the namespace is nil) Or s/he can requests `GET /api/auth?requester_type=ui` and get ui-lived token (the namespace is 'ui'), i will take a look if I can make this a little more clear.